cmake/Intel.cmake

```diff
@@ -36,6 +36,10 @@ else()
         set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -axSSE4.2,CORE-AVX2")
         set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -axSSE4.2,CORE-AVX2")
         set(CMAKE_Fortran_FLAGS_OPT "-no-prec-div -no-prec-sqrt -axSSE4.2,CORE-AVX2")
+    elseif(IVYBRIDGE)
```

Comment:
Rather than specifying a specific Intel architecture, would it be better to use the flags for AVX (that is, the original AVX, not AVX2) here?

Reply:
OK, I've updated switched to using AVX instead.